common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -103,15 +108,27 @@
     private final Context mApplicationContext;
     private final String mActiveBrokerPackageName;
     private final BrokerOperationExecutor mBrokerOperationExecutor;
+    private final HelloCache mHelloCache;
 
     public BrokerMsalController(final Context applicationContext) {
         mApplicationContext = applicationContext;
-        mActiveBrokerPackageName = new BrokerValidator(mApplicationContext).getCurrentActiveBrokerPackageName();
+        mActiveBrokerPackageName = getActiveBrokerPackageName();
         if (TextUtils.isEmpty(mActiveBrokerPackageName)) {
             throw new IllegalStateException("Active Broker not found. This class should not be initialized.");
         }
 
         mBrokerOperationExecutor = new BrokerOperationExecutor(getIpcStrategies(mApplicationContext, mActiveBrokerPackageName));
+        mHelloCache = getHelloCache();
+    }
+
+    @VisibleForTesting
+    public HelloCache getHelloCache() {
```

Comment:
we probably don't need this method....whatever it is doing I think can be done directly in the test

Reply:
This is for testing if the cache is properly wired.. we need this to inject the mock Token cache into BrokerMsalController.